content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+---
+title: Jenkins Configuration as Code (JCasC)
+layout: section
+wip: true
+---
+
+The Jenkins Configuration as Code (JCasC) feature defines Jenkins configuration parameters
+in a human-readable YAML file that can be stored under source code.
+This essentially captures the configuration parameters and values that are used when configuring Jenkins from the web UI.
+The configuration can then be modified by editing this file and then applying it.
+
+[NOTE]
+====
+Traditionally, experienced Jenkins administrators
+created Apache Groovy `init` scripts
+to automate the configuration for their Jenkins instances.
+This works but requires in-depth understanding of Jenkins APIs
+and the ability to write Groovy scripts.
+Such scripts are powerful and can do almost anything,
+but they also provide few protections against configuration errors.
+====
+
+JCasC provides the convenience and flexibility
+of configuring controllers without using the UI.
+It does not require more understanding of the configuration parameters
+than is required to configure Jenkins through the UI
+and it provides some checks on the values that are provided.
+
+The JCasC configuration file can be checked into an SCM,
+which enables you to determine who made what modifications to the configuration
+and to roll back to a previous configuration if necessary.
+
+The
+link:https://plugins.jenkins.io/configuration-as-code[Configuration as Code] plugin
+must be installed on the Jenkins controller
+that you will use to build out your JCasC configuration.
+If you do not see the *Configuration as Code* tile
+in the *System Configuration* section
+of the *Manage Jenkins* page on your dashboard,
+you need to install the plugin.
+
+== Viewing the JCasC File
```

Comment:
The [style guide](https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc#assorted-comments) requests sentence case the "titles" (I interpret that to mean "headings", but that is not a consistent usage throughout the documents, for sure) (optional)
```suggestion
== Viewing the JCasC file
```